Backend/WebApi-PointMap/web.config

```diff
@@ -44,11 +44,15 @@
       <compiler language="vb;vbs;visualbasic;vbscript" extension=".vb" type="Microsoft.CodeDom.Providers.DotNetCompilerPlatform.VBCodeProvider, Microsoft.CodeDom.Providers.DotNetCompilerPlatform, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" warningLevel="4" compilerOptions="/langversion:default /nowarn:41008 /define:_MYTYPE=\&quot;Web\&quot; /optionInfer+" />
     </compilers>

```

Comment:
What are the purpose of these settings you are adding?

Reply:
No idea, it seemed like it was automatically changed, I didn't manually change it

Reply:
please change it back to original if you are not sure why those settings are there, "git pull origin master" might help if your branch is not up to date with master